### PR TITLE
#3221 - add shortcut shift+tab for switching selection mode

### PR DESCRIPTION
**Pull request by StarlaStarla**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

https://github.com/epam/ketcher/assets/35161544/f817db92-7752-42f8-af68-4499116dcaf2

image

## Check list
- [x] unit-tests written
- [x] e2e-tests written
- [x] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"
- [x] reviewers are notified about the pull request

### CODE REVIEW

**Review by chgayane — APPROVED** [review]

**Review by ViktoriaTkacheva — APPROVED** [review]

File: ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  selectNestedTool,
+  SelectTool,
+  selectTool,
+  LeftPanelButton,
+  clickInTheMiddleOfTheScreen,
+} from '@utils';
+
+test.describe('Hot keys', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
```
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts:14]: Could you, please use `waitForPageInit`?

File: ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('select last chosen selected tool when user press ESC', async ({
+    page,
+  }) => {
+    await selectNestedTool(page, SelectTool.FRAGMENT_SELECTION);
+    await selectTool(LeftPanelButton.AddText, page);
+    await page.keyboard.press('Escape');
+    expect(page.getByTestId('select-fragment')).toBeTruthy();
```
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts:28]: I propose to use strict assertions. For example:  
```
await expect(page.getByTestId('select-fragment')).toBeVisible();
```

File: ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts (hunk 1 above)
Comment (Nitvex) [reply ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts:14]: If you use `waitForPageInit`, there is no need to use `await page.goto('')` as it is already inside that function

File: ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-magic-numbers */
```
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts:1]: Can you remove this line? I don't see any magic numbers here

**Review by Nitvex — APPROVED** [review]

File: ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('select last chosen selected tool when user press ESC', async ({
+    page,
+  }) => {
+    await selectNestedTool(page, SelectTool.FRAGMENT_SELECTION);
+    await selectTool(LeftPanelButton.AddText, page);
+    await page.keyboard.press('Escape');
+    expect(page.getByTestId('select-fragment')).toBeVisible();
```
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts:26]: I see these two tests are failing. Could you, please, add `await` for expect? https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible

**Comment by ViktoriaTkacheva** [comment]
Preview. No comments

**Pull request merged**